src/main/java/mate/academy/lib/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.lib;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/init_db";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Dikson_333");

```

Comment:
Consider externalizing database credentials to a configuration file or environment variables to enhance security and prevent sensitive information from being hardcoded in the source code.